setup.py

```diff
@@ -54,14 +54,13 @@ def find_scripts(script_dir, prefix, suffix='.py'):
         'numba',
         'numpy',
         'pandas',
-        'protobuf~=3.20.0',
```

Comment:
why is protobuf gone? not needed anymore with protozfits >=2.5?

Reply:
No need to explicitly list it at least

Reply:
The pin was only necessary because older versions of `protozfits` were incompatible with newer versions of protobuf but didn't specify that. That is long fixed (protozfits correctly pins what versions of protobuf it is compatible with) and newer versions of protozfits are also compatible with the latest protobuf.

Reply:
Since lstchain does not directly use protobuf, it should just rely on the transitive dependency specification of `protozfits`. The pin was only necessary because in the past, that specification was wrong.

Reply:
That's what I though. Thanks for the explanation